bin/updtr

```diff
@@ -1,10 +1,26 @@
 #!/usr/bin/env node
 
+var program = require("commander");
 var updtr = require("../lib");
+var pkg = require("../package.json");
+var reporters = Object.keys(updtr.reporters);
+
+program
+  .version(pkg.version)
+  .description("Update outdated npm modules with zero pain™")
+  .option("-R, --reporter <reporter>", "choose reporter: " + reporters.join(", "), "default");
+
+program.parse(process.argv);
+
+if (!(new RegExp(reporters.join("|")).test(program.reporter))) {
```

Comment:
I think it's rather fragile to use `new RegExp` here. Although it will probably be no problem most of the time, you need to keep in mind that values you pass to the `RegExp` constructor need to be escaped.

I think it's sufficient to do something like:

```javascript
if (program.reporter in updtr.reporters === false) {
```

It's even more readable. :grin: